README.md

```diff
@@ -17,42 +17,13 @@ Aqua.jl provides functions to run a few automatable checks for Julia packages:
 * Check that all external packages listed in `deps` have corresponding
   `compat` entry.
 * `Project.toml` formatting is compatible with Pkg.jl output.
-* There are no "obvious" type piracies ([**new in 0.6**](#notes-on-aqua-06))
+* There are no "obvious" type piracies.
 
 See more in the [documentation](https://juliatesting.github.io/Aqua.jl/dev).
 
-## Quick usage
+## Setup
 
-Call `Aqua.test_all(YourPackage)` from `test/runtests.jl`, e.g.,
-
-```julia
-using YourPackage
-using Aqua
-Aqua.test_all(YourPackage)
-```
-
-## Notes on Aqua 0.6
-
-Aqua 0.6 includes the type piracy detection, thanks to [the PR](https://github.com/JuliaTesting/Aqua.jl/pull/88) by Jakob
-Nybo Nissen (@jakobnissen) and [the original implementation](https://discourse.julialang.org/t/pirate-hunter/20402) by
-Frames Catherine White (@oxinabox).
-
-If this part of Aqua 0.6 causes a trouble, then you can disable the piracy detection
-with a flag as in `Aqua.test_all(YourPackage; piracy = false)`.
-
-## Specifying Aqua version
-
-To avoid breaking test when a new Aqua.jl version is released, it is
-recommended to add version bound for Aqua.jl in `test/Project.toml`:
-
-```toml
-[deps]
-Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-
-[compat]
-Aqua = "0.6"
-```
+Please consult the [documentation](https://juliatesting.github.io/Aqua.jl/dev) for the latest instructions.
```

Comment:
Should this really link to the dev docs, not the stable docs? Or maybe both, with an explanation? (Of course there are also doc links in the badge row at the top, but not everyone might notice them there, so I think it's a good idea to link to the docs in the text, like you do)

Reply:
Changed in 4ff4da36a6261d39a175391a929c6dc0d92658c8